oak/server/rust/oak_loader/src/main.rs

```diff
@@ -199,8 +200,14 @@ fn main() -> anyhow::Result<()> {
         read_to_string(&opt.grpc_tls_private_key).context("could not read gRPC TLS private key")?;
     let grpc_tls_certificate =
         read_to_string(&opt.grpc_tls_certificate).context("could not read gRPC TLS certificate")?;
-    let root_tls_certificate =
-        read_to_string(&opt.root_tls_certificate).context("could not read root TLS certificate")?;
+    let root_tls_certificate = match &opt.root_tls_certificate {
+        Some(certificate_path) if cfg!(feature = "oak_debug") => {
```

Comment:
Does it mean that we can only run CI in debug?

Reply:
Yes.